test/lib/code_corps/github/event/issue_comment_test.exs

```diff
@@ -271,6 +287,11 @@ defmodule CodeCorps.GitHub.Event.IssueCommentTest do
 
         project_ids = project_github_repos |> Enum.map(&Map.get(&1, :project_id))
 
+        project_ids |> Enum.each(fn project_id ->
+          project = Project |> Repo.get_by(id: project_id)
+          insert(:task_list, project: project, inbox: true)
+        end)
```

Comment:
Something to consider, I don't believe in a real use scenario, projects can get created without associated task lists. Maybe we should upgrade our project factory so a set of task lists is always created? As a separate issue, of course.